x-pack/platform/plugins/shared/observability_ai_assistant/public/service/create_chat_service.test.ts

```diff
@@ -83,6 +83,7 @@ describe('createChatService', () => {
     function chat({ signal }: { signal: AbortSignal } = { signal: new AbortController().signal }) {
       return service.chat('my_test', {
         signal,
+        system: 'system',
```

Comment:
Shouldn't this be called `systemMessage` too? 

```suggestion
        systemMessage: 'system',
```

Reply:
I understand the confusion. In the ob-ai-assistant client, the `complete` method uses the argument name `systemMessage` because we will use `system` argument to treat system conversations (distinct concept). However, in the chat service, the argument is named `system` to align with the `inference API`.

It might be better to use `systemMessage` consistently across both `complete` and `chat` methods in ob-ai-assistant for clarity. Naming is hard! :(

Reply:
> the `complete` method uses the argument name `systemMessage` because we will use `system` argument to treat system conversations (distinct concept).

What do you mean with "system conversation"? What is that?